draft-ietf-rats-corim.md

```diff
@@ -175,9 +176,44 @@ The terminology from CBOR {{-cbor}}, CDDL {{-cddl}} and COSE {{-cose}} applies;
 in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
 and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.
 
+### Glossary {#sec-glossary}
+
 This document uses the following terms:
 
 {: vspace="0"}
+
+actual state:
+: Claims that describe a Target Environment instance at a given point in time.
+Endorsed Values and Evidence typically are Claims about actual state.
+An Attester can consist of multiple components, with each component representing a particular scope of appraisal.
+See also ({{Section 2 of -rats-endorsements}}).
+
+Appraisal Claims Set (ACS):
+: A structure that holds ECTs that have been appraised.
```

Comment:
```suggestion
: A structure that holds ECTs ( for ECTs see below) that have been appraised.
```

Reply:
If you look through the entries, this is not the only one.
There are constant references between the glossary entries, even 1:n.